packages/portfolio-contract/tools/portfolio-actors.ts

```diff
@@ -344,14 +344,25 @@ export const planDepositTransfers = (
   return transfers;
 };
 
-export const planTransfer = (
+type MakeGasEstimator = (chainName: AxelarChain) => {
+  getWalletEstimate: () => Promise<bigint>;
+  getFactoryContractEstimate: () => Promise<bigint>;
+  getReturnFeeEstimate: () => Promise<bigint>;
+};
+
+export const planTransfer = async (
   dest: PoolKey,
   amount: NatAmount,
   feeBrand: Brand<'nat'>,
-): MovementDesc[] => {
+  makeGasEstimator: MakeGasEstimator,
+  accountIdByChain: Record<string, `${string}:${string}:${string}`>,
+): Promise<MovementDesc[]> => {
   const { protocol: p, chainName: evm } = PoolPlaces[dest];
   const steps: MovementDesc[] = [];
 
+  const padding = 3n; // padding multiplier for gas estimates
```

Comment:
Multiplying by 3 seems like a lot, and bigint multiplication may prove too limiting (e.g., what if we settle on increasing by 50%?). Suggestion: convert to a function.
```suggestion
  const increaseFeeEstimate = (estimate: bigint): bigint => estimate * 3n;
```

Reply:
When I suggested 3x, I didn't mean 3x whatever the Axelar API comes up with. I mean: 3x whatever the actual cost is. I gather that the Axelar API may already have up to a 20x factor, so this 3x wouldn't be necessary at all.

Reply:
the 20x discrepancy was only observed in EVM -> Agoric estimates. for our regular Agoric -> EVM estimates, the estimates was ~1x therefore i added the  "* 3" to only those txs